handlers/admin/adminRolesPage.go

```diff
@@ -27,5 +27,5 @@ func AdminRolesPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	data := Data{Roles: roles}
-	handlers.TemplateHandler(w, r, "admin/rolesPage.gohtml", data)
+	handlers.TemplateHandler(w, r, "rolesPage.gohtml", data)
```

Comment:
@jules, this page must be in `admin/` as it's admin move the page rather than change this if required.